validation/spec.json

```diff
@@ -60,7 +56,7 @@
 
 	"type": "object",
 	"patternProperties": {
-		"^[a-zA-Z]+$": {
+		"^[a-zA-Z0-9]+$": {
```

Comment:
just a comment: The XML schema allows generic CDATA for names, so something like `hello_world_my_channel` is also possible (much to my annoyance since RooFit uses `_` also as separator in its function names).. so we should be aware of that with an eye towards ROOT HF compatibility, but i'm fine with extending this regex on a when-needed basis

Reply:
Noted.